src/App.test.jsx

```diff
@@ -20,12 +22,17 @@ const user = {
 }
 
 // 🚨 Create your server
+const server = setupServer(
+  rest.get(`${process.env.REACT_APP_SUPABASE_URL}/rest/v1/users`, (req, res, ctx) =>
+    res(ctx.json([user]))
+  )
+)
 
 // 🚨 Listen for server start
-beforeAll()
+beforeAll(() => server.listen())
 
 // 🚨 Close server when complete
-afterAll()
+afterAll(() => server.close())
```

Comment:
We will want to have our first mock accessible globally. We can accomplish this by placing these functions into the setupTests.js folder.